oshdb/src/main/java/org/heigit/bigspatialdata/oshdb/index/XYGrid.java

```diff
@@ -32,12 +32,10 @@
  * <tr>
  * <td></td><td></td><td colspan="3">-90 lat</td><td></td><td></td><td></td>
  * </tr>
- * </table>
- * <br>
- * Longitude +180 will be wrapped around to -180. Coordinates lying on
- * grid-borders will be assigned to the north-eastern cell.
- *
+ * </table></p>
```

Comment:
Is `<table>` in `<p>` supposed to be right? I know it wasn't allowed in the past ([HTML 4](https://www.w3.org/TR/html401/struct/text.html#h-9.3.1)), but it might be allowed in [HTML 5](https://www.w3.org/TR/2018/WD-html53-20181018/grouping-content.html#the-p-element). I'm not really aware of the current rules there, the documentation says: _Some block elements cannot be children of p elements._

Reply:
AFAIK, javadoc requires one to put `<p>` after every empty line / before every paragraph. At least checkstyle complains otherwise:

> Empty line should be followed by <p> tag on the next line. (16:0) [JavadocParagraph]

Reply:
But javadoc also complains about the unnecessary `</p>` (compare [log](https://jenkins.ohsome.org/blue/organizations/jenkins/oshdb/detail/PR-251/3/pipeline#step-39-log-213)). I think you should setting the `</p>` before `<table>` would be the clean way to do it.